### DISCUSSION TITLE
支付宝体验版无法打开页面，只出现tabbar，提示 【页面遇到了一点问题】

### DISCUSSION THREAD

**Discussion by fengchaseyou** [NervJS/taro#11897] [Q & A]
显示如下：
![图片](https://user-images.githubusercontent.com/37128857/172345168-610f0cfe-c650-4b4c-bdce-c5fe901ed367.png)

**Comment by fengchaseyou** [comment]
@Chen-jj

**Comment by fengchaseyou** [comment]
@Chen-jj  大佬在吗，有知道这种情况吗

**Reply by TheKonka** [reply]
你这个 只有一张图片看不出啥呀，连控制台报错都没有，建议直接给个demo仓库
